mdto.py

```diff
@@ -424,21 +441,61 @@ def to_xml(self) -> ET.ElementTree:
                 "xsi:schemaLocation": "https://www.nationaalarchief.nl/mdto https://www.nationaalarchief.nl/mdto/MDTO-XML1.0.1.xsd",
             },
         )
-    
+
         root = ET.SubElement(mdto, "informatieobject")
 
         # allow users to pass either a single IdentificatieGegevens object, or a list thereof
         if isinstance(self.identificatie, IdentificatieGegevens):
             self.identificatie = [self.identificatie]
-            
+
         for i in self.identificatie:
             root.append(i.to_xml("identificatie"))
 
+        if self.naam:
+            naam_elem = ET.SubElement(root, "naam")
```

Comment:
Volgens mij bestaat `self.naam` altijd, dus deze `if` is in principe overbodig. 

Ik neig er naar om hem misschien weg te laten. Ik heb liever dat python een error gooit als `naam`/verplichte gegevens ontbreken dan dat men informatieobjecten zonder naam aanmaakt (wat dus al min of meer onmogelijk is, maar misschien dat het je met wat duwen en trekken lukt). 

Dit past bij mijn onderliggende filosofie van dit programma: _als_ er een XML bestand wordt aangemaakt, dan is er een zeer sterke garantie (~100%) dat dat valide MDTO is. Als je deze `if` er als zodanig inlaat dan vervalt deze garantie een beetje. 

Als je toch liever deze fout expliciet wilt catchen zou ik ergens een `else` neerzetten